nmostesting/ControllerTest.py

```diff
@@ -397,7 +397,7 @@ def _create_sender_json(self, sender):
         if "sdp_params" in sender:
             # Mapping sdp_params names to sender names
             # For JPEG XS Senders, 'bit_rate' and 'st2110_21_sender_type' MUST be included
-            param_mapping = {'video_bit_rate': 'bit_rate', 'video_type_parameter': 'st2110_21_sender_type',
+            param_mapping = {'bit_rate': 'bit_rate', 'TP': 'st2110_21_sender_type',
                              'packet_transmission_mode': 'packet_transmission_mode'}
```

Comment:
IS-04 `packet_transmission_mode` corresponds to _two_ SDP format-specific parameters, `packetmode` (which is required and currently hard-coded in the _video-jxsv.sdp_ template) AND `transmode` (which is optional with a default value, which with the currently hard-coded `packetmode` corresponds to the `codestream` value we're using everywhere)...

Does this matter? I'm not sure... I guess I'm just worried if we appear to be supporting variable values, we need more code to map between them, but we probably *don't* need that right now, so maybe the code should make clear this/those params are more or less hardcoded...

Reply:
Yes, OK good point - I'll add a comment to explain the situation.